reports/views/products/inexpensiveproducts.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from django.shortcuts import render
+from bangazonapi.models import Product
+from reports.views import Connection
+
+
+def inexpensiveproduct_list(request):
+    """Function to build an HTML report of products over $1000"""
+    if request.method == 'GET':
+        # connect to database
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+
+        # query all products over $1000
+        db_cursor.execute("""
+              SELECT 
+                *
+              FROM bangazonapi_product
+              WHERE bangazonapi_product.price <= 999 
```

Comment:
What if the cost is 999.50?

Reply:
I changed the SQL statement to <1000 that was a good point and I didn't take that into account.